assets/js/amp-admin-tables.js

```diff
@@ -0,0 +1,46 @@
+/* global jQuery, ampAdminTables */
+( function( $ ) {
+	'use strict';
+
+	var adminTables = {
```

Comment:
Some of the code here might be duplicated with https://github.com/Automattic/amp-wp/blob/develop/assets/src/amp-validation-error-detail-toggle.js

Can we unify them?

Reply:
@westonruter  I started working on this but realized a dependency of the script you mentioned is `wp-dom-ready` which is a part of Gutenberg. So this script doesn't load for me since on my test site Gutenberg is not activated. Are we assuming that everyone will have Gutenberg as well or why is this a dependency? Or did I miss something in the build process?

Reply:
Hi @westonruter and @jacobschweitzer,
Do you think we should add `dom-ready` to [package.json](https://github.com/Automattic/amp-wp/blob/e63a879c6047ae5a10fdd6d0ea028b83eb37a42f/package.json#L12), like how `i18n` is included there:

```json
  "@wordpress/dom-ready": "^2.0.0",
  "@wordpress/i18n": "^1.1.0",

```

Still, `npm run build` seemed to build [amp-validation-error-detail-toggle.js](https://github.com/Automattic/amp-wp/blob/21413ba755917c6d6888c98fcc2ac5608e118266/assets/src/amp-validation-error-detail-toggle.js) fine in my local, and that script ran without error on the 'AMP Validation Error Index' page.

Reply:
@kienstra I've added that line to `package.json` and ran `npm run build` / `npm run dev` but the script still won't load on the Error Index page. Am I missing something here? I did have to install webpack and wp-cli but I have those now so I'm not sure if there is another dependency I'm missing since there aren't any errors in the build process now.